generator/src/main/java/org/apache/kafka/message/MessageGenerator.java

```diff
@@ -267,10 +283,10 @@ public static void main(String[] args) throws Exception {
         if (args.length == 0) {
             System.out.println(USAGE);
             System.exit(0);
-        } else if (args.length != 3) {
+        } else if (args.length != 4) {
```

Comment:
Why we need separate exit code for mis-used arguments?

Reply:
It's traditional for invoking a program with no arguments to print out the usage message, and exit with success, which is 0.  If there was a usage error, then we exit with failure, which is 1.

Reply:
Sg